src/Statistics/WilsonBinomialConfidenceIntervalEstimatorTests.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "Statistics/BinomialConfidenceInterval.h"
+
+namespace orbit_statistics {
+
+const WilsonBinomialConfidenceIntervalEstimator kEstimator;
```

Comment:
I would prefer to instantiate this object in each test separately to ensure that there aren't any unintended side effects due to sharing this object.